alicloud/resource_alicloud_polardb_database.go

```diff
@@ -0,0 +1,187 @@
+				ValidateFunc: validation.StringInSlice(POLAR_DB_CHARACTER_SET_NAME, false),
+				Optional:     true,
+				Default:      "utf8",
+				ForceNew:     true,
+			},
+
+			"account_name": {
+				Type:     schema.TypeString,
+				ForceNew: true,
+				Optional: true,
+			},
+
+			"account_privilege": {
+				Type:         schema.TypeString,
+				ValidateFunc: validation.StringInSlice(POLAR_DB_ACCOUNT_PRIVILEGE_NAME, false),
+				Default:      "ReadWrite",
+				ForceNew:     true,
+				Optional:     true,
+			},
+
+			"db_description": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+		},
+	}
+}
+
+func resourceAlicloudPolarDBDatabaseCreate(d *schema.ResourceData, meta interface{}) error {
+
+	client := meta.(*connectivity.AliyunClient)
+	polarDBService := PolarDBService{client}
+	request := polardb.CreateCreateDatabaseRequest()
+	request.RegionId = client.RegionId
+	request.DBClusterId = d.Get("db_cluster_id").(string)
+	request.DBName = d.Get("db_name").(string)
+	request.CharacterSetName = d.Get("character_set_name").(string)
+	request.AccountName = d.Get("account_name").(string)
+	request.AccountPrivilege = d.Get("account_privilege").(string)
+
+	if v, ok := d.GetOk("db_description"); ok && v.(string) != "" {
+		request.DBDescription = v.(string)
+	}
+
+	if inst, err := polarDBService.DescribePolarDBCluster(request.DBClusterId); err != nil {
+		return WrapError(err)
+	} else if inst.Engine == string(PostgreSQL) {
+		return WrapError(Error("At present, it does not support creating 'PostgreSQL' database. Please login DB instance to create."))
+	}
+
+	err := resource.Retry(5*time.Minute, func() *resource.RetryError {
+		raw, err := client.WithPolarDBClient(func(polarDBClient *polardb.Client) (interface{}, error) {
+			return polarDBClient.CreateDatabase(request)
+		})
+		if err != nil {
+			if IsExceptedErrors(err, OperationDeniedDBStatus) {
+				return resource.RetryableError(err)
+			}
+			return resource.NonRetryableError(err)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		return nil
+	})
+
+	if err != nil {
+		return WrapErrorf(err, DefaultErrorMsg, d.Id(), request.GetActionName(), AlibabaCloudSdkGoERROR)
+	}
+
+	d.SetId(fmt.Sprintf("%s%s%s", request.DBClusterId, COLON_SEPARATED, request.DBName))
+
+	return resourceAlicloudPolarDBDatabaseRead(d, meta)
+}
+
+func resourceAlicloudPolarDBDatabaseRead(d *schema.ResourceData, meta interface{}) error {
+	client := meta.(*connectivity.AliyunClient)
+	polarDBService := PolarDBService{client}
+	object, err := polarDBService.DescribePolarDBDatabase(d.Id())
+	if err != nil {
+		if NotFoundError(err) {
+			d.SetId("")
+			return nil
+		}
+		return WrapError(err)
+	}
+	parts, err := ParseResourceId(d.Id(), 2)
+	if err != nil {
+		d.SetId("")
+		return nil
+	}
+	d.Set("db_cluster_id", parts[0])
+	d.Set("db_name", object.DBName)
+	d.Set("character_set_name", object.CharacterSetName)
+	d.Set("db_description", object.DBDescription)
+
```

Comment:
why missing account parameters?

Reply:
> why missing account parameters?

I remove input parameters about account，because create account should be in account moudle